mix.exs

```diff
@@ -19,6 +20,11 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [licenses: ["Apache 2.0"],
+    links: %{"GitHub" => "https://github.com/elixir-lang/ex_doc"}]
```

Comment:
@eksperimental That was my initial proposal, but, to end the discussion and move on with something more fun I've deleted everything that reference to contributors :laughing: